### ISSUE TITLE
[Improvement][Filter chip] Apply high contrast theme to filter chip (selected)

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#494]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

.

### OS Version

.

### App version

.

### Describe the issue

Like it's done on radio button, switch and checkbox, high contrast black color should be applied also to filter chip in selected state

### Expected Behavior

.

**Comment by nouha06** [comment]
@pya35 is this only for the selected state? What about the pressed state? In contrast mode, the color is currently orange, but it should be black.

**Comment by pya35** [comment]
@nouha06 pressed state is not checked for accessibility matters. We care only about initial, hover and focus states.

**Comment by pya35** [comment]
OK
